db/schema.rb

```diff
@@ -182,6 +182,11 @@
     t.string "postcode"
     t.string "reachskills_org_name"
     t.string "reachskills_op_link"
+    t.string "role_description"
+    t.string "skills_needed"
+    t.string "when_volunteer_needed"
+    t.string "contact_details"
+    t.string "about_organization"
```

Comment:
If we are getting this from the organisation description on the show page then we will not need this field on the table